From: "Jeremy T. Bouse" <Jeremy.Bouse@UnderGrid.net>
Date: Mon, 15 Feb 2016 13:00:19 -0500
Subject: Localize generated documentation

Remove external resource references from the Sphinx layout to
generate local documentation to package under /usr/share/doc for
the paramiko-doc.
---
 sites/shared_conf.py | 5 +----
 1 file changed, 1 insertion(+), 4 deletions(-)

diff --git a/sites/shared_conf.py b/sites/shared_conf.py
index 99fab31..343b579 100644
--- a/sites/shared_conf.py
+++ b/sites/shared_conf.py
@@ -10,10 +10,7 @@ extensions = ['alabaster', 'sphinx.ext.intersphinx']
 html_theme = 'alabaster'
 html_theme_options = {
     'description': "A Python implementation of SSHv2.",
-    'github_user': 'paramiko',
-    'github_repo': 'paramiko',
-    'analytics_id': 'UA-18486793-2',
-    'travis_button': True,
+    'github_button': False,
 }
 html_sidebars = {
     '**': [
